### PR TITLE
Simplifies the example code for glossary entries of boundary_id and manifold_id

### PR DESCRIPTION
**Pull request by krishnakumarg1984**
Fixes #9320 

I had to close the other PR due to getting too confused with git branch/merge issues. Have addressed the reviewer comments in this PR

### CODE REVIEW

File: doc/doxygen/headers/glossary.h [hunk 1]
```diff
@@ -327,14 +327,10 @@
...
  * @code
- *   for (typename Triangulation<dim>::active_cell_iterator
- *          cell = triangulation.begin_active();
- *        cell != triangulation.end();
- *        ++cell)
- *     for (unsigned int f=0; f<GeometryInfo<dim>::faces_per_cell; ++f)
- *       if (cell->face(f)->at_boundary())
- *         if (cell->face(f)->center()[0] == -1)
- *           cell->face(f)->set_boundary_id (42);
+ *   for ( auto &face : triangulation.active_face_iterators() )
```
Comment (bangerth) [inline doc/doxygen/headers/glossary.h]: Can you remove the whitespace?
```suggestion
 *   for (auto &face : triangulation.active_face_iterators())
```

File: doc/doxygen/headers/glossary.h [hunk 2]
```diff
@@ -1273,9 +1269,7 @@
  * center has an $x$ component less than zero:
  *
  * @code
- * for (typename Triangulation<dim>::active_cell_iterator cell =
- *  triangulation.begin_active();
- *  cell != triangulation.end(); ++cell)
+ * for ( auto &cell : triangulation.active_cell_iterators() )
```
Comment (bangerth) [inline doc/doxygen/headers/glossary.h]:
```suggestion
 * for (auto &cell : triangulation.active_cell_iterators())
```

**Comment by krishnakumarg1984** [comment]
Done

**Comment by bangerth** [comment]
Thanks for making the changes. Would you be willing to squash your commits again? As a general rule, we almost always squash a sequence of commits into a set of self-contained units. Here, there is no reason to leave this as 3 commits -- 1 should be enough.

**Comment by bangerth** [comment]
@krishnakumarg1984 -- ping?

**Comment by tamiko** [comment]
rebased and squashed the PR

**Pull request merged**